includes/menu.inc

```diff
@@ -119,3 +119,46 @@ function oe_bootstrap_theme_add_dropbutton_classes(array &$links): void {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_menu_local_tasks().
+ */
+function oe_bootstrap_theme_preprocess_menu_local_tasks(array &$variables): void {
+  // Sort local tasks by weight.
+  uasort($variables['primary'], [
+    '\Drupal\Component\Utility\SortArray',
+    'sortByWeightProperty',
+  ]);
+  uasort($variables['secondary'], [
+    '\Drupal\Component\Utility\SortArray',
+    'sortByWeightProperty',
+  ]);
+
+  foreach ($variables['primary'] as $link) {
```

Comment:
Please, consider if could be good to add a function like getLinks that receives string (primary or secondary) and returns the array mounted. The code of both foreach is almos 95% equal.